jekyll/_cci2/local-cli.md

```diff
@@ -7,170 +7,311 @@ categories: [troubleshooting]
 order: 10
 ---
 
-This document provides instructions for installing and using the CircleCI CLI. 
-
-* TOC
-{:toc}
-
 ## Overview
-{:.no_toc}
 
-The `circleci` command-line interface enables you to reproduce the CircleCI environment locally and run jobs as if they were running on the hosted application for more efficient debugging and configuration in the initial setup phase.
+The CircleCI CLI is a command line interface that leverages many of CircleCI's
+advanced and powerful tools from the comfort of your terminal. Some of the
+things you can do with the CircleCI CLI include: 
 
-You can also run `circleci` commands in your [`.circleci/config.yml`]({{ site.baseurl }}/2.0/configuration-reference/) file for jobs that use the primary container image. This is particularly useful for globbing or splitting tests among containers.
+- Debug and validate your CI config 
```

Comment:
all list items should use same punctuation, so either remove the one period, or add a period to them all